detox/test/e2e/29.webview.test.js

```diff
@@ -269,14 +287,43 @@ describe('Web View', () => {
     });
   });
 
+  describe(':ios: inner frame', () => {
+    /** @type {Detox.WebViewElement} */
+    let webview;
+    const mockServer = new MockServer();
+
+    beforeAll(async () => {
+      mockServer.init();
+
+      if (device.getPlatform() === 'android') {
+        // Android needs to reverse the port in order to access the mock server
+        await device.reverseTcpPort(mockServer.port);
```

Comment:
not necessary at the moment, since this is iOS only suite, but I prefer to leave it here for forward compatibility.